people/compensation.md

```diff
@@ -2,17 +2,32 @@
 
 ## Compensation philosophy
 
-See [the 2i2c Careers page](https://2i2c.org/jobs/) for a short description of our compensation philosophy.
-
-Salaries at 2i2c are determined solely by job title as well as several steps within that job (corresponding to experience in a particular job).
-We have [internal salary documents for the specific positions and levels](https://docs.google.com/spreadsheets/d/1FJM5pAbc0EWhu4CpPjlbWTMOsZAnivEd2ZBIZIdwpE8/edit?usp=sharing).
-
 :::{seealso}
 We are [working on an update of our compensation policy](https://docs.google.com/document/d/1Ixkg-YgeTO-Xc9neQHIlTHa7psyhlFHQQCSfCr-K0cQ/edit#).
 See that document for ideas that are more specific / structured than this page.
 :::
 
-### Define compensation levels with GuideStar
+:::{admonition} Aspirational
+We are a young organization, and this page describes the salary policy that we aim to enact in the coming years.
+It may not accurately reflect our current practices, but we share it for transparency and to provide a goal for our team to shoot for.
+:::
+
+Our salary policy should ensure that our team feels financially supported and respected in their roles, and should be aligned with our values and mission as a non-profit organization.
+Team members should feel that 2i2c is a competitive workplace from a salary perspective, while recognizing that we do not have the financial resources of most tech companies.
+Below are a few guiding principles of our salary policy:
+
+- **Benchmarked to non-profit tech**. We should benchmark our salaries based on similar roles in our sector (non-profits that do technical or research work, depending on the role). See [](compensation:salaries:guidestar) for the current process we follow.
+  - If we determine that a sector is systemically under-paying a given role in a way that makes us uncompetitive, then we may increase the salary band for the role to be more aligned with the private sector.
+- **Explicit rationale**. All of our salaries must be tied to explicit roles, with consistent and transparent responsibilities and expectations for entering and advancing a role.
+- **Transparent compensation by role**. Our salaries, benefits, and the roles that they are attached to should be available to all.
+- **Fixed salaries for a role**. We do not negotiate salaries within a given role.
+  Salaries are fixed for a given role and tenure[^tidelift].
+- **Equal pay for equal work**. We practice [equal pay for equal work](https://en.wikipedia.org/wiki/Equal_pay_for_equal_work) across our team. Salaries do not distinguish between geographic location or cost of living.
```

Comment:
It would be useful to state that we do this by benchmarking against a high cost of living area, rather than a low one